contracts/SnappAuction.sol

```diff
@@ -26,7 +26,20 @@ contract SnappAuction is SnappBase {
     mapping (uint16 => StandingOrderData) public standingOrders;
 
     uint public auctionIndex = MAX_UINT;
-    mapping (uint => SnappBaseCore.PendingBatch) public auctions;
+
+    struct PendingAuction {
+        address solver;
+        uint objectiveValue;           // Traders utility
+        bytes32 solutionHash;          // Succinct record of trade execution & prices
+        uint solutionAcceptedTime;   // Time solution was accepted (written at time of solutionHash)
+        bytes32 tentativeState;        // Proposed account state during bidding phase
+        uint16 size;                   // Number of orders in this auction
+        bytes32 shaHash;               // Rolling shaHash of all orders
```

Comment:
nit: let's call this order hash?

Reply:
Actually looking at this now, it might be equally good to reuse the PendingBatch struct nested as you suggested over slack. Up to you. Having nested structs could serve as a more structured documentation. Something like

```
struct PendingAuction {
  PendingBatch orderBatch,
  SolutionBid bestSolution
  uint solutionRevealedTime
}
```

Reply:
I'll give it a look! Turns out there is a loss of coverage because of a new if branch that I need to deal with anyway.

Reply:
Unfortunately we can't have nested structs at this point (as it is still Experimental). I got this

```
The following types are only supported for getters in the new experimental ABI encoder: struct SnappBaseCore.PendingBatch memory. Either remove "public" or use "pragma experimental ABIEncoderV2;" to enable the feature.
```

Reply:
So then I will be calling it `orderHash` as suggested.